packages/tools/devtools/devtools-example/src/ClientUtilities.ts

```diff
@@ -35,6 +35,17 @@ export interface ContainerInfo {
 	appData: AppData;
 }
 
+<<<<<<< HEAD
+function initializeTinyliciousClient(logger?: ITelemetryBaseLogger): TinyliciousClient {
+	console.log(`Initializing Tinylicious client...`);
+	return new TinyliciousClient({
+		connection: {
+			port: 443,
+			domain: "https://alexvy86-super-doodle-j677467qg2pwvp-7070.preview.app.github.dev",
+		},
+		logger,
+	});
```

Comment:
This is the kind of change needed on an app that uses TinyliciousClient. The URL should match the one used in the updates to `server/routerlicious/packages/tinylicious/src/resourcesFactory.ts`.

When I originally did this, devtools-example worked like that, but we've since moved it to use the more low-level approach of directly creating a loader, so this change doesn't work in this app anymore but I kept the merge conflict here to show what needs to be done.